packages/allure-playwright/package.json

```diff
@@ -32,6 +32,7 @@
     "typescript": "^4.4.2"
   },
   "dependencies": {
+    "@types/node": "^16",
```

Comment:
@playwright/test already depends on @types/node, so I'd move it to a devDependency instead.